### PR TITLE
Merge with squash

### PR DESCRIPTION
**Pull request by mkowalski**
This is a rebase with following squashes performed

```
5b219d5 manifests: update references from 4.13 to 4.14
2924cbcb UPSTREAM: <carry>: Updating openshift-kubernetes-nmstate-handler images to be consistent with ART
b29ee80d UPSTREAM: <carry>: Updating openshift-kubernetes-nmstate-operator images to be consistent with ART
f1abcfcc OPNET-261: Integrate the ART build of nmstate-console-plugin image in the CSV
bf073596 manifests: update io_nmstates.yaml
43df0b60 move to stable channel - part 2
d4bc01a3 OPNET-256: Move from version-specific to stable channel
769469ae UPSTREAM: <carry>: Updating openshift-kubernetes-nmstate-operator images to be consistent with ART
bb1a4b6d UPSTREAM: <carry>: Updating openshift-kubernetes-nmstate-handler images to be consistent with ART
8b51d9ff update manifest for 4.13
2d12ff90 UPSTREAM: <carry>: run make ocp-update-bundle-manifests
62cdefa5 UPSTREAM: <carry>: Add empty spec to NMState CR example
c8b6e3df UPSTREAM: <carry>: Remove NNCP example from manifests
ee874612 UPSTREAM: <carry>: add ipv4 example to the interfaces in the alm-examples
7819bc85 UPSTREAM: <carry>: Update Dockerfile images to match ART
61ee11d9 UPSTREAM: <carry>: Update Dockerfile images to match ART
4f7b97d6 UPSTREAM: <carry>: Update Dockerfile images to match ART
77df634d UPSTREAM: <carry>: Remove handler SA permissions from CSV
190f76a8 UPSTREAM: <carry>: Update bundle manifests after rebase
f24a70e9 UPSTREAM: <carry>: Update Dockerfile images to match ART
b6182064 UPSTREAM: <carry>: Update OPM for 4.12
ac2a2f4f UPSTREAM: <carry>: Make sure yq v4 is installed for bundle updates
b29545d9 UPSTREAM: <carry>: Add manifests for 4.12
0f857a9e UPSTREAM: <carry>: Add make target to update bundle manifests and install bundle
e14a4eca UPSTREAM: <carry>: Remove old manifests
cc7b3cdb UPSTREAM: <carry>: Fix OLM skipRange replacement
da70e89c UPSTREAM: <carry>: Update CSV after rebase
a6c70faa UPSTREAM: <carry>: Update Dockerfile images to match ART
33c48f55 UPSTREAM: <carry>: Update NMState CRD
4e0d9fea UPSTREAM: <carry>: Update 4.11 CSV file after rebase
ed3c3d13 UPSTREAM: <carry>: add priority class to operator CSV
465ae226 UPSTREAM: <carry>: Add qosClass to operator CSV
024a069f UPSTREAM: <carry>: Update Dockerfile images to match ART
6aab3952 UPSTREAM: <carry>: Add manifests for 4.11
2f519488 UPSTREAM: <carry>: Add operators app label in CSV
d9085387 UPSTREAM: <carry>: Update Dockerfile images to match ART
ba282198 UPSTREAM: <carry>: Remove initialization resource from CSV
943a83a6 UPSTREAM: <carry>: enable all archs in nmstate example CR
67e42409 UPSTREAM: <carry>: Add disconnected annotation for OperatorHub
607f4fda UPSTREAM: <carry>: Fix JSON error in initialization resource
26bcd49e UPSTREAM: <carry>: Add ppc64le and s390x to 4.10 CSV
f5c01ec8 UPSTREAM: <carry>: add arm64 to 4.10 CSV
ba9425f0 UPSTREAM: <carry>: Add 4.10 release OLM manifests
97f3643f UPSTREAM: <carry>: Provide default NMState CR
9b504085 UPSTREAM: <carry>: Remove serviceaccount manifest
9f5c002c UPSTREAM: <carry>: Update dockerfile to match ART
e0ae9773 UPSTREAM: <carry>: update naming convention
93b48ef3 UPSTREAM: <carry>: Define the suggested namespace for NMState CR
1ed45114 UPSTREAM: <carry>: Update Dockerfile images to match ART
e12cac14 UPSTREAM: <carry>: Add manifests to .PHONY in Makefile
15238288 UPSTREAM: <carry>: Add manifests for 4.9
bb95b06b UPSTREAM: <carry>: Add manifests for 4.8
40f1e21b UPSTREAM: <carry>: Add manifests for 4.7
62021187 UPSTREAM: <carry>: Updating openshift-kubernetes-nmstate-operator images to be consistent with ART
```

```
92f28cce UPSTREAM: <carry>: Add mko as reviewer and approver
1d239b10 UPSTREAM: <carry>: Add dougsland as an approver
d7d674e2 UPSTREAM: <carry>: Remove yboaron and creydr from OWNERS
fa88ab9d UPSTREAM: <carry>: Update OWNERS
97feffdf UPSTREAM: <carry>: Add midstream approvers and reviewers
```

```
c295349 hack: fix git log to use correct order
8eb855ae ocp-rebase: do not use static branch names
3f518d98 ocp-rebase: general improvements
4018f526 UPSTREAM: <carry>: ocp-rebase: use -s in cherry-pick
19a59123 UPSTREAM: <carry>: Add helper script for rebase
```

### CODE REVIEW

**Comment by mkowalski** [comment]
/hold

I merged another PR in between so need to revise this anyway. But before doing so I want us all to agree on how the squash should look like. This one here is my arbitrary choice which may or may be not good

**Comment by dougsland** [comment]
@mkowalski let us know when it's ready to merge please.

**Comment by mkowalski** [comment]
I need to redo it as we merged stuff in between, but it shouldn't take long

**Comment by dougsland** [comment]
LGTM

**Comment by dougsland** [comment]
@mkowalski it's flying...

File: manifests/bases/kubernetes-nmstate-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -18,17 +18,17 @@ metadata:
     categories: OpenShift Optional
     description: |
       Kubernetes NMState is a declaritive means of configuring NetworkManager.
-    containerImage: quay.io/openshift/origin-kubernetes-nmstate-operator:4.13
+    containerImage: quay.io/openshift/origin-kubernetes-nmstate-operator:4.14
```
Comment (cybertron) [inline manifests/bases/kubernetes-nmstate-operator.clusterserviceversion.yaml:21]: I'm curious why these changes are showing up in the PR diff since they shouldn't be a change from what the previous tree looked like. Maybe just the ordering of the PR merges?
